### PR TITLE
DAR-17 - Python CI

### PR DESCRIPTION
**Pull request by Ninja243**
Branch removes some 3rd party assets, switches to twine for pushing to PyPI and adds release info to the readme.

Note, the `DETA_SDK_TEST_PROJECT_KEY` and `DETA_SDK_TEST_BASE_NAME` keys need to be set for the test stage to pass.

https://linear.app/detonians/issue/DAR-17/python-sdk-cd

### CODE REVIEW

**Comment by Ninja243** [comment]
Having a weird time with some jobs being cancelled, looks like it might be due to the container not having enough memory to do these at the same time -> https://github.com/actions/virtual-environments/issues/1918

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
+        with:
+          python-version: ${{ matrix.python-version }}
+      # Run install script
+      - name: Install dependencies
+        run: "scripts/install"
       # Lint the code
-      - name: Python linters Black Check
-        uses: jose-lpa/python-black-check-action@1.0.0
+      - name: Lint code with black
+        run: python3 -m black .
```
Comment (yankeexe) [inline .github/workflows/pull_request.yml]: We can update this line to: 

```bash 
$ make format
```

https://github.com/deta/deta-python/blob/54d329e9a4a6dd8eee0be3534197a025a48c2353/Makefile#L19

Comment (Ninja243) [reply .github/workflows/pull_request.yml]: You're right! Thanks

File: deta/__init__.py [hunk 2]
```diff
@@ -16,11 +16,10 @@
 except Exception:
     pass
 
+__version__ = 0.4
```
Comment (abdelhai) [inline deta/__init__.py]: the current version is `0.7`. also change it in setup.py, for some reason we didn't commit it there too

File: scripts/test_publish [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+if [ -d 'venv' ] ; then
+    PREFIX="venv/bin/"
+else
+    PREFIX=""
+fi
+
+if [ ! -z "$GITHUB_ACTIONS" ]; then
+  git config --local user.email "action@github.com"
+  git config --local user.name "GitHub Action"
```
Comment (abdelhai) [inline scripts/test_publish:13]: is this recommended? don't we need to have our own details?

**Review by abdelhai — CHANGES_REQUESTED** [review]

File: deta/__init__.py (hunk 2 above)
Comment (Ninja243) [reply deta/__init__.py]: Will do!

File: scripts/test_publish (hunk 3 above)
Comment (Ninja243) [reply scripts/test_publish:13]: We can do that, that just seems to make sure that the tags and releases are attributed to the Github bot!

Comment (Ninja243) [reply scripts/test_publish:13]: @abdelhai is there a specific username/email combo you'd like associated with those actions? :)

**Review by abdelhai — APPROVED** [review]

**Pull request merged**